sdk/src/main/java/com/mapbox/maps/viewannotation/ViewAnnotationOptionsExt.kt

```diff
@@ -0,0 +1,46 @@
+@file:JvmName("ViewAnnotationOptionsExt")
```

Comment:
we don't want to expose it, this should be removed

Reply:
It's needed in test.

Reply:
do we need it in the Java test? still don't get it...

Reply:
`ViewAnnotationOptions.frame()` below is internal and should be accessible without specifying `@file:JvmName("ViewAnnotationOptionsExt")` I assume. Specifying it that way you do now actually add an empty entry in `sdk/api/metalava.txt` file above which we do not need